{
  "comments": [
    {
      "key": {
        "uuid": "5ad0253b_57b63e3e",
        "filename": "python-oslo-cache.spec",
        "patchSetId": 1
      },
      "lineNbr": 193,
      "author": {
        "id": 128
      },
      "writtenOn": "2019-01-22T14:02:07Z",
      "side": 1,
      "message": "this else means that we are not building python3 so this directory will not exist in that case. Anyway, it has -f so will not fail.",
      "range": {
        "startLine": 193,
        "startChar": 0,
        "endLine": 193,
        "endChar": 55
      },
      "revId": "701385697a50bdcb0a02cca8daf7375001e5c413",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    }
  ]
}